packages/compiler/core/scanner.ts

```diff
@@ -1031,6 +1054,19 @@ export function createScanner(
     }
   }
 
+  function unescapeBacktickedOne(pos: number): string {
+    const ch = input.charCodeAt(pos + 1);
+    switch (ch) {
+      case CharCode.Backslash:
+        return "\\";
+      case CharCode.Backtick:
+        return "`";
+      default:
```

Comment:
Again discussion is pending, but I am leaning towards allowing the same thing in quotes and backticks so that it is always possible to format quoted member to backticked identifier and truly consider the quoted member syntax back compat only.

Reply:
Updated. Now backticks allows the same thing in quotes, in addition, the backtick "`" itself.